x-pack/plugins/ml/public/ui_actions/create_swim_lane.tsx

```diff
@@ -41,6 +42,28 @@ export function createAddSwimlanePanelAction(
         getIconType: () => PLUGIN_ICON,
       },
     ],
+    order: 10,
```

Comment:
Could you put this to `40` so that it goes at the top. This is the most widely used of the ML embeddables so it makes sense for it to go at the top I think.